src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/Teams/Table/actions.html.twig

```diff
@@ -1,5 +1,6 @@
 <td class="actions teams__actions">
-    <div tabindex="0">
+    <div tabindex="0" class="">
```

Comment:
empty class attribute here, that's likely something which can be removed

Reply:
Empty class has been removed

Reply:
has been removed